src/esp32/pin.rs

```diff
@@ -1,13 +1,10 @@
-use esp_idf_hal::gpio::{OutputMode, Pin, PinDriver};
+use esp_idf_hal::gpio::{Pin, PinDriver};
```

Comment:
Why are we removing output modes of pins here? Seems like it's more complete than what we have in rdk